public/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I'm not sure if this worked for you? Typically when you want to add a whole directory to a .gitignore, you will put a `/` at the end of the file name. So I would use `node_modules/`